Topic1/src/com/topic1/IndividualOffer.java

```diff
@@ -0,0 +1,24 @@
+package com.topic1;
+
+public class IndividualOffer implements Offer {
+
+	Item item;
```

Comment:
[High] Why there is an *Item* and *IndividualOffer*? There should be just an Item (or product) and an Offer (a group of items and/or Offers)